python/pyarrow/tests/test_extension_type.py

```diff
@@ -388,3 +389,24 @@ def test_parquet(tmpdir, registered_period_type):
     pa.unregister_extension_type(period_type.extension_name)
     result = pq.read_table(filename)
     assert result.column("ext").type == pa.int64()
+
+
+def test_to_numpy():
+    period_type = PeriodType('D')
+    storage = pa.array([1, 2, 3, 4], pa.int64())
+    arr = pa.ExtensionArray.from_storage(period_type, storage)
+
+    expected = storage.to_numpy()
+
+    # TODO waiting on ARROW-6749 PR to enable to_numpy
+    # result = arr.to_numpy()
+    # np.testing.assert_array_equal(result, expected)
```

Comment:
This actually needs https://github.com/apache/arrow/pull/5718 to be merged (if that PR gets merged first, will update it here)

Reply:
You can re-enable it now :-)